lib/fake_braintree/client_assets.rb

```diff
@@ -0,0 +1,48 @@
+require 'capybara'
+require 'capybara-webkit'
+require 'fake_braintree/asset_versions'
+require 'net/http'
+
+module FakeBraintree
+  class ClientAssets
+    URL = 'https://js.braintreegateway.com/v2/braintree.js'
+
+    def initialize
+      @asset_versions = FakeBraintree::AssetVersions.new
+    end
+
+    def save
+      if client_version == @asset_versions.client_version
+        puts 'Client assets up to date'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Do not write to stdout. Use Rails' logger if you want to log.